charms/knative-serving/requirements-unit.in

```diff
@@ -1,5 +1,23 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+# Please note this file introduces dependencies from the charm's requirements.in,
+# special attention must be taken when updating this or the other .in file to try
+# to avoid incompatibilities.
+# Rules for editing this file:
+#   * Removing a dependency that is no longer used in the unit test file(s)
+#     is allowed, and should not represent any risk.
+#   * Adding a dependency in this file means the dependency is directly used
+#     in the unit test files(s).
+#   * ALL python packages/libs used directly in the unit test file(s) must be
+#     listed here even if requirements.in is already adding them. This will
+#     add clarity to the dependency list.
```

Comment:
Same comment as [this](https://github.com/canonical/knative-operators/pull/125/files#r1275298592)